.github/goodtables_report.py

```diff
@@ -43,8 +46,43 @@
 ERROR_FILE = os.environ['ERROR_FILE']
 REPO = os.environ['GITHUB_REPOSITORY']
 ERROR_MAX = int(os.environ['ERROR_MAX'])
+SCHEMA_PATH = os.environ['SCHEMA_PATH']
+
+def is_invalid_schema():
+    """Send a Slack message and return true if the datapackage json is invalid."""
+
+    try:
+        with open(os.environ['SCHEMA_PATH']) as f:
+            json.load(f)
+    except Exception as err:
+        j_payload = {
+            'text': "There was an error loading the JSON:",
+            'attachments': [
+                {
+                    "fallback": f'```{str(err)}```',
+                    "color": "danger",
+                    "fields":[{
+                       "title": os.environ['SCHEMA_PATH'],
+                       "value": f'```{str(err)}```'
+                    }]
+                }
+            ]
+        }
+
+        # post stringified payload to url
+        requests.post(os.environ['SLACK_GOODTABLES_WEBHOOK'], json.dumps(j_payload))
+        if os.environ.get('SLACK_GOODTABLES_WEBHOOK_2'):
+            requests.post(os.environ['SLACK_GOODTABLES_WEBHOOK_2'], json.dumps(j_payload))
+
+        return True
+    
+    return False
+
 
 def main():
+    if is_invalid_schema():
```

Comment:
Do we have to check this every time? Isn't there something you can key on in the validation output to determine if validation actually ran?